packages/ui-charts/assets/civic-logo-animated-invert.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 23.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 338.8 153" style="enable-background:new 0 0 338.8 153;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#EE495C;}
+	.st1{fill:#FFFFFF;stroke:#FFFFFF;stroke-width:0.24;stroke-linejoin:round;stroke-miterlimit:1.4142;}
+	.st2{fill:none;stroke:#EE495C;stroke-width:16.5;stroke-linejoin:round;stroke-miterlimit:1.4142;}
+	.st3{fill:#EE495C;stroke:#EE495C;stroke-width:0.24;stroke-linejoin:round;stroke-miterlimit:1.4142;}
+</style>
+<g id="logo-set">
+	<path class="st0" d="M107,34.4h16.1V101H107V34.4z"/>
+	<path class="st0" d="M138.9,34.4h16.8l10.7,40.7l10.5-40.7h17L173.5,101h-14.2L138.9,34.4L138.9,34.4z"/>
+	<path class="st0" d="M212.1,34.4h16.1V101h-16.1V34.4z"/>
+	<path class="st0" d="M56.9,80.2c-7-7-7-18.1,0-24.9c3.4-3.4,7.8-5.1,12.6-5.1c4.7,0,9.2,1.9,12.6,5.1l11.7-11.7
+		c-6.4-6.4-15.1-10-24.3-10s-17.8,3.6-24.3,10c-13.4,13.4-13.4,35.1,0,48.3c6.8,6.6,15.4,10,24.3,10s17.5-3.4,24.3-10L82,80.2
+		C75.1,87,63.9,87,56.9,80.2z"/>
+	<path class="st1" d="M264.5,80.2c-7-7-7-18.1,0-24.9c3.4-3.4,7.8-5.1,12.6-5.1c4.7,0,9.2,1.9,12.6,5.1l11.7-11.7
+		c-6.4-6.4-15.1-10-24.3-10s-17.8,3.6-24.3,10c-13.4,13.4-13.4,35.1,0,48.3c6.8,6.6,15.4,10,24.3,10s17.5-3.4,24.3-10l-11.7-11.7
+		C282.7,87,271.5,87,264.5,80.2z"/>
+	<path class="st2" d="M295.6,85.9c-8.8,9-25.4,11.5-37.4-0.4c-6.5-6.4-12.5-22.9,0.8-36.5"/>
+	<path class="st3" d="M328.1,122.7h-40.7L272,138l-15.3-15.3h-29H10.6V10.6L0.1,0.1v133h252.2l19.7,19.8l19.8-19.8h46.8L328.1,122.7
+		L328.1,122.7z"/>
+	<path class="st1" d="M0.1,0.1l10.5,10.5h317.5v112.1l10.5,10.3V0.1C338.6,0.1,0.1,0.1,0.1,0.1z"/>
+</g>
+</svg>
```

Comment:
Do these SVGs also need to be here if they are in `ui-brand`? Is the intent to keep `ui-charts` from being bloated by superfluous `ui-brand` content?

Reply:
Apparently there was a whole unused assets folder in `ui-charts` being useless...